Add stored NFT data Bubblegum section

danenbm opened the pull request:
Link to preview of specific page I updated: https://developer-hub-git-danenbm-stored-nft-metaplex.vercel.app/bubblegum/stored-nft-data

MarkSackerberg commented:
Looking good to me.
Shouldn't we link the different reference implementations you are explaining in the list though? Either to their docs (which there are none currently i assume) or their github repositories?

febo commented on src/pages/bubblegum/stored-nft-data.md:
```diff
@@ -4,4 +4,72 @@ metaTitle: Bubblegum - Storing and Indexing NFT Data
 ---
 
-_Coming soon..._
+As mentioned in the [Overview](/bubblegum#read-api), whenever compressed NFTs (cNFTs) are created or modified, the corresponding transactions are recorded on-chain in the ledger, but the cNFT state data is not stored in account space.  This is the reason for the massive cost savings of cNFTs, but for convenience and usability, the cNFT state data is indexed by RPC providers and available via the  ...
```
Should this say "Metaplex Read API" and link to the public repo?

danenbm replied:
Renamed it to "Metaplex Read API" here and where it is introduced in the Overview page (index.md).

danenbm commented:
> Looking good to me. Shouldn't we link the different reference implementations you are explaining in the list though? Either to their docs (which there are none currently i assume) or their github repositories?

I wrote some additional text explaining how different RPC providers use some or all of our reference code, and added the link to our Metaplex reference implementation.

I also added a link to the RPC provider page, which shows the different RPCs who support the Metaplex standard, and has links to their detailed API documentation.  I think since they do have docs we want to be linking to those, rather than to github repos.

Let me know what you think.  Thanks!

febo reviewed (approved):
LGTM
